src/services/layouts/handlers.js

```diff
@@ -1,8 +1,10 @@
 module.exports = {
   GET: app => async (request, reply) => {
+    const offset = request.query.offset ? parseInt(request.query.offset, 10) : 0
+    const limit = request.query.limit ? parseInt(request.query.limit,  10) : 10
     return app.mongo.db
       .collection('layouts')
-      .find()
+      .find().skip(offset).limit(limit)
```

Comment:
Nitpick!

Just move the skip and limit function to individual lines like done in the rest of the project.